plugins/package-managers/bazel/src/test/kotlin/BazelTest.kt

```diff
@@ -22,32 +22,13 @@ package org.ossreviewtoolkit.plugins.packagemanagers.bazel
 import io.kotest.core.spec.style.WordSpec
 import io.kotest.engine.spec.tempdir
 import io.kotest.matchers.collections.beEmpty
-import io.kotest.matchers.shouldBe
 import io.kotest.matchers.shouldNot
 
 import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
 import org.ossreviewtoolkit.model.config.RepositoryConfiguration
 import org.ossreviewtoolkit.utils.test.getAssetFile
 
 class BazelTest : WordSpec({
-    "transformBazelVersion()" should {
-        "remove everything except for the version number" {
-            val bazelVersionOutput = """
-                Bazelisk version: development
```

Comment:
I even put the Bazelisk output in the test, even though I didn't use it in the Docker image. Surprising that this ever worked. 😅